applications/debug/unit_tests/resources/unit_tests/subghz/mastercode2_raw.sub

```diff
@@ -0,0 +1,6 @@
+Filetype: Flipper SubGhz RAW File
```

Comment:
delete the file it doesn't work, too little data recorded

Reply:
Deleted in then new commit